KD_Lib/KD/vision/CSKD/sampler.py

```diff
@@ -0,0 +1,238 @@
+import csv, torchvision, numpy as np, random, os
+from PIL import Image
+
+from torch.utils.data import (
+    Sampler,
+    Dataset,
+    DataLoader,
+    BatchSampler,
+    SequentialSampler,
+    RandomSampler,
+    Subset,
+)
+from torchvision import transforms, datasets
+from collections import defaultdict
+
+
+class PairBatchSampler(Sampler):
+    def __init__(self, dataset, batch_size, num_iterations=None):
+        self.dataset = dataset
+        self.batch_size = batch_size
+        self.num_iterations = num_iterations
+
```

Comment:
docstrings are missing for many functions.

Reply:
Will add it